ckanext/data_qld_theme/templates/scheming/form_snippets/update_due.html

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
If this is an exact copy of the scheming date form snippet just call the snippet directly.
eg. {%- snippet 'scheming/form_snippets/date.html', field=field, data=data, errors=errors -%}